### ISSUE TITLE
Doesn't handle some movies as expected

### ISSUE THREAD

**Issue by orbitalflower** [stefanodvx/crunpyroll#7]
With the Crunchyroll beta, some movies consist of a single video and do not appear to have seasons or episodes that the API can handle. For example:

`GRK5Z4P96` (Black Jack the Movie)
`GRWE20Z3R` (Cardcaptor Sakura the Movie)

Attempting to call `get_series()` or `get_seasons()` with one of these IDs returns this error:

`crunchyroll_beta.errors.CrunchyrollError: [404] Error occured: Invalid series identifier`

Calling `get_episodes()` on the ID returns an empty list. It's not clear how to get from a series ID to an episode ID. It might make sense for the module to generate a dummy season and episode listing just for consistency.

**Comment by willhoh** [comment]
I also had a mistake. But it looks like they are no longer available on Crunchyroll. Hence the 404.

**Comment by SuperSonicHub1** [comment]
I also ran into this issue, it's not that it isn't available, but that you're using the wrong endpoint. Here's a snippet ([Unlicensed](https://unlicense.org)) that should help you out:
```python
from crunchyroll_beta import Crunchyroll
from crunchyroll_beta.types import StreamsInfo

def get_movie(cr: Crunchyroll, movie_id: str) -> List[StreamsInfo]:
	"""Solves https://github.com/stefanodvx/crunpyroll/issues/7"""
	res = cr._make_request(
		"GET",
		f'https://beta-api.crunchyroll.com/content/v2/cms/movie_listings/{movie_id}/movies',
		params=dict(locale=cr.locale)
	)
	return [
		StreamsInfo(cr._make_request('GET', item['playback']))
		for item in res.get('data', [])
```

**Comment by stefanodvx** [comment]
Fixed in new 2.0 version.
